ci/jenkins/test-vm.sh

```diff
@@ -202,7 +206,7 @@ function create_kubeconfig_files {
     kubectl config --kubeconfig=${WORKDIR}/${ANTREA_AGENT_KUBECONFIG} use-context antrea-agent@kubernetes
 
     # Kubeconfig to access AntreaController
-    ANTREA_API_SERVER_IP=$(kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 != role {print $6}')
+    ANTREA_API_SERVER_IP=$(kubectl get nodes -o wide --no-headers=true | awk -v role="$CONTROL_PLANE_NODE_ROLE" '$3 = role {print $6}')
```

Comment:
Neither the preivous code nor the current one is correct. It must be `==`. How did it work?

Reply:
Maybe this solution is more accurate?
```
ANTREA_API_SERVER_IP=$(kubectl  get pods -nkube-system -l component=antrea-controller -ojson | jq -r .items[0].status.podIP)
```

Reply:
change it to `==` works. `=` happened to work for the ci environment because it is a single node cluster.